src/pyenphase/envoy.py

```diff
@@ -250,6 +252,7 @@ async def _request(
                 timeout=self._timeout,
             )
 
+        request_end = time.time()
```

Comment:
Please use `time.monotonic()` since it will always produce sane results if ntp runs and wall clock time moves backwards

Reply:
implemented.